### PR TITLE
Support for part-id fallback

### PR DESCRIPTION
**Pull request by afkiwers**
added support for part-id fallback. if activated it will also look at part name. This would cover the case where users don't have inventree ids but MPNs. Those could then be used to find the right part.

fixes: https://github.com/afkiwers/inventree_kicad/issues/63

### CODE REVIEW

File: inventree_kicad/KiCadLibraryPlugin.py [hunk 1]
```diff
@@ -262,34 +269,44 @@ def import_meta_data(self, request):  # noqa
...
+
                 except Part.DoesNotExist:
-                    logger.debug(f'Part ID: {inventree_id} does not belong to an existing part, skipping')
-                    continue
+                    invalid_part = True
+
+                    # try also the part name if user wants it
+                    if self.get_setting('IMPORT_INVENTREE_ID_FALLBACK', None):
+                        try:
+                            part = Part.objects.get(name=inventree_part_id)
```
Comment (atanisoft) [inline inventree_kicad/KiCadLibraryPlugin.py:298]: shouldn't inventry_part_id be updated here since it may be used for `PartAttachment` below?

Comment (atanisoft) [reply inventree_kicad/KiCadLibraryPlugin.py:298]: or use `part=part` below.

Comment (afkiwers) [reply inventree_kicad/KiCadLibraryPlugin.py:298]: Fair point, thanks

Comment (afkiwers) [reply inventree_kicad/KiCadLibraryPlugin.py:298]: fixed!

**Pull request merged**